jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -740,8 +748,8 @@ public Response getInvoicePayments(@PathParam("accountId") final UUID accountId,
     @Produces(APPLICATION_JSON)
     @Consumes(APPLICATION_JSON)
     @Path("/{accountId:" + UUID_PATTERN + "}/" + INVOICE_PAYMENTS)
-    @ApiOperation(value = "Trigger a payment for all unpaid invoices")
-    @ApiResponses(value = {@ApiResponse(code = 204, message = "Successful operation"),
+    @ApiOperation(value = "Trigger a payment for all unpaid invoices", response = InvoiceJson.class, responseContainer = "List")
+    @ApiResponses(value = {@ApiResponse(code = 201, message = "Successful operation"),
```

Comment:
We should update Slate docs.

Reply:
> We should update Slate docs.

Yes, but not quite ready

I have some second thoughts, I think we should keep the `204` for when there was no payment made, similar to what we do [here](https://github.com/killbill/killbill/blob/master/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/InvoiceResource.java#L383). Also, currently, if no invoice were paid the `Location` link returned is wrong as it would create a follow GET that would fetch *all* invoices

Reply:
Fixed in https://github.com/killbill/killbill/pull/1433/commits/e01dd5cea57974231a38c781b1a5fa972bda2ff7